fs-sim/CHANGELOG.md

```diff
@@ -32,8 +34,8 @@
 ### Patch
 
 * Make it build with `ghc-9.12`.
-* Support `io-classes-1.8`.
 * Drop support for `ghc-8.10` and `ghc-9.2`.
```

Comment:
This still says that support for `ghc-9.2` is dropped

Reply:
Thanks, fixed